view/main/logo.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" fill="currentcolor" class="main_logo-icon" viewBox="0 0 192 76"><path d="M88 76V32c0-6.75-1.72-12.22-5-16.22l5.29-5.29A10.6 10.6 0 0 0 94 12a2 2 0 0 0 0-4c-3.81 0-6-2.19-6-6a2 2 0 1 0-4 0 10.6 10.6 0 0 0 1.51 5.66L80.22 13c-4-3.28-9.47-5-16.22-5H40c-6.75 0-12.22 1.72-16.22 5l-5.29-5.34A10.6 10.6 0 0 0 20 2a2 2 0 1 0-4 0c0 3.81-2.19 6-6 6a2 2 0 1 0 0 4 10.6 10.6 0 0 0 5.66-1.51L21 15.78c-3.28 4-5 9.47-5 16.22v44zM64 40a10 10 0 1 1 0 20 10 10 0 0 1 0-20m-24 0a10 10 0 1 1 0 20 10 10 0 0 1 0-20"/><path d="M40 56a6 6 0 1 0 0-12 6 6 0 0 0 0 12m0-10a4 4 0 1 1 0 8 4 4 0 0 1 0-8m24 10a6 6 0 1 0 0-12 6 6 0 0 0 0 12m0-10a4 4 0 1 1 0 8 4 4 0 0 1 0-8m88 10a6 6 0 0 0 5.543-3.704 6.005 6.005 0 0 0-1.3-6.539 6.002 6.002 0 0 0-9.232.91A6.002 6.002 0 0 0 152 56m0-10a4 4 0 1 1 0 8 4 4 0 0 1 0-8m-24 10a6 6 0 0 0 5.543-3.704 6.005 6.005 0 0 0-1.3-6.539 6.002 6.002 0 0 0-9.232.91A6.002 6.002 0 0 0 128 56m0-10a4 4 0 1 1 0 8 4 4 0 0 1 0-8"/><path d="M176 76V63.8a10 10 0 0 0 0-19.6V32c0-15.25-8.75-24-24-24h-24c-15.25 0-24 8.75-24 24v12.2a10 10 0 0 0-5.753 16.128A10 10 0 0 0 104 63.8V76zm0-27.66a6 6 0 0 1 0 11.32zm-72 11.32a6 6 0 0 1 0-11.32zm24 .34a10.002 10.002 0 0 1-9.808-11.95 9.998 9.998 0 0 1 13.635-7.289A10 10 0 0 1 138 50a10 10 0 0 1-10 10m24 0a10.002 10.002 0 0 1-9.808-11.95 9.998 9.998 0 0 1 13.635-7.289A10 10 0 0 1 162 50a10 10 0 0 1-10 10m-42-30v-2.41l5.93 6 6.07-6 6 6 6-6 6 6 6-6 6 6 6-6 6 6 6-6v2.82l-5.93 6-6.07-6-6 6-6-6-6 6-6-6-6 6-6-6.05-6 5.9-6-6.21z"/></svg>
```

Comment:
This is the same svg as before. I just remove `class="main_logo-path"` because change color with `fill="currentcolor"` on root element svg is easier smaller for css specificity

 See this lines
 https://github.com/evilmartians/oklch-picker/pull/157/files#diff-3a57e01122d7154eb0a5de8d23798ab965ba9d02f0c28e259d4c278a51aa9429R113-R114